### PR TITLE
Adding Work Breakdown Structures Point

### PR DESCRIPTION
**Pull request by harrythomas1998**

### CODE REVIEW

**Review by alinakazzaa — COMMENTED** [review]
A reference to a resource about WBS for further reading would be a good addition

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
...
+
+## **Task Estimation**
+
+### **Work Breakdown Structures**
+
+**WBS** (Work Breakdown Structure) describes what tasks need to be completed in a project. However, it does not include factors such as timelines or resources. The goal of the WBS is to focus the team on what has to be done.
+
+When your breaking down what has to be done it is important to remove verbs from each level. You are not describing the tasks necessary to complete what has to be done, you are just describing them.
+
+![WBS Diagram](bdt.png)
```
Comment (mikeygillen) [inline README.md]: Move picture to the bottom and scale down if possible

File: README.md [hunk 2]
```diff
@@ -1 +1,19 @@
...
+# **Assignment**
+
+
+## **Task Estimation**
+
+### **Work Breakdown Structures**
+
+**WBS** (Work Breakdown Structure) describes what tasks need to be completed in a project. However, it does not include factors such as timelines or resources. The goal of the WBS is to focus the team on what has to be done.
+
+When your breaking down what has to be done it is important to remove verbs from each level. You are not describing the tasks necessary to complete what has to be done, you are just describing them.
```
Comment (chriskavanagh97) [inline README.md]: you're**

File: README.md [hunk 3]
```diff
@@ -1 +1,19 @@
...
+
+![WBS Diagram](bdt.png)
+
+Some of the benefits of WBS include:
+*   **Project Schedule:** Once you define all the tasks required to complete the project, as well as their hierarchical relationships, it will be much easier to assign resources and set deadlines.
+*   **Accountability:** Since all elements in a WBS are mutually exclusive, it helps create accountability. A team assigned to a single work package is wholly accountable for its completion. This reduces overlaps in responsibility.
```
Comment (chriskavanagh97) [inline README.md]: solely instead of wholly

**Review by chriskavanagh97 — COMMENTED** [review]
good

**Pull request closed without merging**